server/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 5000;
+const mongoose = require('mongoose');
+const passport = require('passport');
+
+const morgan = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+require('./models/Note');
+
+const keys = require('./config/keys');
+mongoose.connect(keys.mongoURI);
+
+require('./services/passport')(passport);
+
```

Comment:
All good here, just a style consideration for grouping script set-up. Some linter configurations _require_ it, so something to consider for readability and flow down the page: putting all your requirement declarations first, e.g.
```js
const express = require('express');
const mongoose = require('mongoose');
const passport = require('passport');
const morgan = require('morgan');
const cookieParser = require('cookie-parser');
const bodyParser = require('body-parser');
const session = require('express-session');
const keys = require('./config/keys');
require('./models/Note');
require('./services/passport')(passport);

const app = express();
const PORT = process.env.PORT || 5000;
mongoose.connect(keys.mongoURI);
```

From there sometimes I arrange them alphabetically, or maybe on their importance/order of use. Just a though for ya - this is otherwise just fine 👍 I did test this on my side (after setting up mongodb localhost and commenting out the config file keys - no hoisting issues with parsing, compiling and running your script with these changes, however, with this style guidelne, there may be times when hoisting and relative placement of requirement HOCs may take precedence.